db/seeds.rb

```diff
@@ -1,14 +1,17 @@
 require 'faker'
 
+user = FactoryBot.create(:user, role: :research_group)
+
 6.times do |n|
 	title = Faker::Lorem.sentences(number: 1)
 	body = Faker::Lorem.paragraph
-	author = Faker::Name.name
-
+  author = Faker::Name.name
+  
 	Article.create!(
 		title: title,
 		body: body,
-		author: author,
+    author: author,
+    user: user,
```

Comment:
What is the difference between author and user in this case?

Reply:
Author is an attribute of the Article, user is the creator of the article